packages/litexa/src/parser/litexa.pegjs

```diff
@@ -917,22 +917,22 @@ pronounce "tomato" as "<phoneme alphabet="ipa" ph="/təˈmɑːtoʊ/">tomato</pho
 Note that you must use SSML in the replacement text; the Litexa
 SSML shorthand statements (e.g "<!Bravo>") will not work.
 
-### localized pronunciations
+### Localized Pronunciations
 
 If you plan to publish your skill to multiple locales, you can define
 locale-specific pronunciations for your voiced responses. Simply add
 your locale-specific pronunciation definitions to the Litexa files that
-reside in said locale's Litexa project directory.
+reside in said locale's Litexa project directory (see the Localization
+chapter for further information).
 
-::: tip Use a pronounce.litexa file in each locale's Litexa project directory
-If defining locale-specific pronunciations, we recommend creating a
-pronounce.litexa file in each of your locale's Litexa project directories.
-Put all of your locale-specific pronunciation definitions inside of it.
+::: tip Tip
```

Comment:
Good to know about auto-header. But just as well, searching through the codebase, there seems to be inconsistent uses:

1. Currently see a lot of both `:::tip` and `::: tip`.
2. Auto-header is all uppercase, but most of our manual uses are regular-case headings.

I'll remove the space, but keep the manual heading so that it is regular casing.

Reply:
Ah, right - forgot the auto-header is fully upper case.

If the spacing after the `:::` is inconsistent anyhow, go ahead and disregard this comment and leave as is.